_source/logzio_collections/_metrics-sources/service-bus.md

```diff
@@ -0,0 +1,165 @@
+
+---
+
+To monitor your Azure service metrics,
+we recommend configuring your services
+to send their metrics to Azure Monitor.
+When you set up Metricbeat using the configuration on this page,
+Metricbeat will collect metrics from Azure ServiceBuss
+and forward them to [Logz.io](http://logz.io/).
+
+#### Metricbeat setup
+
+**Before you begin, you'll need**:
+
+* [Metricbeat 7.6](https://www.elastic.co/guide/en/beats/metricbeat/7.6/metricbeat-installation.html),
+* [Azure CLI 2.0](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) or higher
+
+<div class="tasklist">
+
+##### Log in to Azure with the CLI
+
+Run this command in the command line:
+
+```yml
+az login
+```
+
+To complete the process, sign in through your browser.
+
+##### Get your subscription details
+
+Run this command:
+
+```yml
+az account show | grep '"id"\\|"tenantId"'
+```
+
+The response shows the subscription ID (the `id` field)
+and tenant ID (the `tenantId` field).
+You'll need this information later on, so paste it in your text editor.
+
+###### Sample response
+
+```
+"id": "d94b1fba-0289-557e-b063-0b6bfc1bdca0",
+"tenantId": "9ae0715a-0689-56e8-bb88-2b22f1fa7299",
+```
+
+##### Create a new Azure AD application
+
+The Metricbeat configuration needs to include credentials for an AD application with reader permissions.
+
+If you don't already have an AD application with reader permissions,
+run this command to create one.
+Replace `<<SUBSCRIPTION-ID>>` with the `id` value from step 2:
+
+```yml
+az ad sp create-for-rbac --role reader \\
+--scopes /subscriptions/<<SUBSCRIPTION-ID>> \\
+-n logzio-metricbeat \\
+| grep '"appId"\\|"password"'
+```
+
+The response shows the client ID (the `appId` field)
+and client secret (the `password` field).
+You'll need this information later on, so paste it in your text editor.
+
+###### Sample response
+
+```yml
+"appId": "3dcdf53e-f93f-5902-8df2-235c8635aa4d",
+"password": "e6ab6d24-4907-5d11-a132-a171ef55355d",
+```
+
+##### Download the [Logz.io](http://logz.io/) public certificate
+
+
+
+For HTTPS shipping, download the [Logz.io](http://logz.io/) public certificate to your certificate authority folder.
+You'll need to run this command on the server that hosts Metricbeat:
+
+```yml
+sudo curl <https://raw.githubusercontent.com/logzio/public-certificates/master/TrustExternalCARoot_and_USERTrustRSAAAACA.crt> --create-dirs -o /etc/pki/tls/certs/COMODORSADomainValidationSecureServerCA.crt
+```
+
+##### (_Optional_) Disable the system module
+
+By default, Metricbeat ships system metrics from its host.
+Disable this module so you don't unintentionally send host metrics:
+
+```yml
+sudo metricbeat modules disable system
+```
+
+##### Add [Logz.io](http://logz.io/) configuration
+
+Now you'll set up the Metricbeat
+to collect metrics from Azure ServiceBus.
+
+You'll need to replace the values surrounded by angle brackets
+`<< >>`
+using the parameters below the code block. 👇
+
+```yml
+metricbeat.modules:
```

Comment:
![image](https://user-images.githubusercontent.com/6851078/86256759-5fe2c200-bbc1-11ea-8771-d99bc07a5efa.png)

Reply:
There are certain differences. Are they critical?

Reply:
Thanks for finding this!